falco.yaml

```diff
@@ -773,6 +775,39 @@ metrics:
   convert_memory_to_mb: true
   include_empty_values: false
 
+  ###############################################
+# Falco driver selection                    #
+###############################################
+
+# [Stable] `driver_mode`
+#
+# --- [Description]
+#
+# Falco supports different driver modes for capturing syscall events. The choice
+# of driver mode can significantly impact the performance and compatibility of
+# Falco with your system. Choose the appropriate driver mode based on your
+# system's configuration and requirements.
+#
+# Available driver modes:
```

Comment:
I would avoid the word `driver`. Moreover, in this list we still miss:
* the capability to read `scap-file` so something like ` mode: replay`
* the capability to open a source plugin, but this is not so easy since we have also the `load_plugin` config

Reply:
I'm not totally convinced about this. The `driver` word expresses the need for an external running component to the instrument to collect the data. On the other hand, a plugin is a component that runs together with `falco` in the same process. A scap file is an entirely different use case. Moreover, I believe the `replay` mode should only be available from the command line as a flag (why one should configure it in `falco.yaml` :thinking: ?).

 Likely, your proposal tried to mimic the _scap open_ behavior. However, we should refrain from exposing how scap open works to the end users and keep the _driver_ distinct from other modes (as it is now), IMO.

Reply:
I'd say: `source` instead of `driver`, and `source.mode: plugin` and deprecate `load_plugin` in next release; for now it will be supported along side (ie: `load_plugin` would override the `source.mode`.